servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractTestRest.java

```diff
@@ -737,7 +737,7 @@ public void commitLogPagingAndFilteringByAuthor() throws BaseNessieClientServerE
             .collect(Collectors.toList());
     List<CommitMeta> completeLog =
         StreamingUtil.getCommitLogStream(
-                api, branch.getName(), null, null, null, OptionalInt.of(pageSizeHint))
+                api, branch.getName(), null, null, null, OptionalInt.of(pageSizeHint), false)
```

Comment:
Could we have an overloaded method in `StreamingUtil` with the default value for the new parameter to reduce the amount of changes in callers?

Reply:
it is overall 3 places in entire code. So may be no need of overloading ?

Reply:
fair enough